crates/collab/src/rpc.rs

```diff
@@ -2286,25 +2289,6 @@ where
     Ok(())
 }
 
-async fn forward_find_search_candidates_request(
```

Comment:
While auditing usages of `forward_request`, I noticed this had the exact same implementation as `forward_read_only_project_request`, and so removed it as a drive by improvement.